packages/api-key-stamper/src/tink/elliptic_curves.ts

```diff
@@ -172,7 +172,7 @@ export function pointDecode(point: Uint8Array): JsonWebKey {
     point.length !== uncompressedLength
   ) {
     throw new Error(
-      "Invalid length: point is not in compressed or uncompressed format"
+      "Invalid API key: Ensure that you are using a valid public and private key for your API key"
```

Comment:
Suggestion: 

This is def a more helpful message than the old one, but maybe we could include some of the old stuff because it points the user in the direction of the fact that the formatting of the API key is what is off. 

Something like: 

`Invalid API key: Ensure that you are using a valid public and private key for your API key, provided in either the compressed or uncompressed point format`

Reply:
I've incorporated some of the original error message in the new message